novels.sql

```diff
@@ -0,0 +1,114 @@
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Arthur", "Miller");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Alexandre", "Dumas");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Arthur Conan",  "Doyle");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Robert Louis",  "Stevenson");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Fyodor", "Dostoyevsky");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Agatha", "Christie");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Ray", "Bradbury");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("George", "Orwell");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("H.G.", "Wells");
+INSERT INTO authors (nameFirst, nameLast) VALUES ("Chinua", "Achebe");
+
+INSERT INTO novels (title, authorId) VALUES ("Dracula", 1);
+INSERT INTO novels (title, authorId) VALUES ("The Picture of Dorian Gray", 2);
+INSERT INTO novels (title, authorId) VALUES ("The Color Purple", 3);
+INSERT INTO novels (title, authorId) VALUES ("War and Peace", 4);
+INSERT INTO novels (title, authorId) VALUES ("A Tale of Two Cities", 5);
+INSERT INTO novels (title, authorId) VALUES ("The Crucible", 6);
+INSERT INTO novels (title, authorId) VALUES ("The Three Musketeers", 7);
+INSERT INTO novels (title, authorId) VALUES ("The Hound of the Baskervilles", 8);
+INSERT INTO novels (title, authorId) VALUES ("The Strange Case of Dr. Jekyll and Mr. Hyde", 9);
+INSERT INTO novels (title, authorId) VALUES ("Crime and Punishment", 10);
+INSERT INTO novels (title, authorId) VALUES ("Murder on the Orient Express", 11);
+INSERT INTO novels (title, authorId) VALUES ("Fahrenheit 451", 12);
+INSERT INTO novels (title, authorId) VALUES ("Animal Farm", 13);
+INSERT INTO novels (title, authorId) VALUES ("The Time Machine", 14);
+INSERT INTO novels (title, authorId) VALUES ("Things Fall Apart", 15);
+
+INSERT INTO genres (name) VALUES ("Fiction,Horror,Fantasy");
+INSERT INTO genres (name) VALUES ("Fiction,Horror,Gothic,Fantasy");
+INSERT INTO genres (name) VALUES ("Fiction,Historical Fiction");
+INSERT INTO genres (name) VALUES ("Fiction,Historical Fiction,War,Russian Literature");
+INSERT INTO genres (name) VALUES ("Fiction,Historical Fiction");
+INSERT INTO genres (name) VALUES ("Fiction,Historical Fiction");
+INSERT INTO genres (name) VALUES ("Fiction,Historical Fiction,Adventure,French Literature");
+INSERT INTO genres (name) VALUES ("Fiction,Mystery,Crime,Thriller");
+INSERT INTO genres (name) VALUES ("Fiction,Mystery,Science Fiction,Horror");
+INSERT INTO genres (name) VALUES ("Fiction,Russian Literature,Mystery");
+INSERT INTO genres (name) VALUES ("Fiction,Mystery");
+INSERT INTO genres (name) VALUES ("Fiction,Science Fiction,Dystopia");
+INSERT INTO genres (name) VALUES ("Fiction,Science Fiction,Dystopia");
+INSERT INTO genres (name) VALUES ("Fiction,Science Fiction,Time Travel");
+INSERT INTO genres (name) VALUES ("Fiction,Historical Fiction,African Literature");
```

Comment:
You definitely didn't understand normalization or many-to-many relationships. You need to go back and review that content. You wouldn't store a genre like `Fiction` over and over. You'd store it once and use the relationship to link it to all novels it is associated with.